sub-projects/hub/src/app/config.jsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import { HashRouter, Route, Redirect } from 'react-router-dom';
+import axios from 'axios';
+
 import LoginPage from './components/auth/login-page.jsx';
+import AddAdmin from './components/admin/add-admin.jsx';
 
+axios.defaults.withCredentials = true;
```

Comment:
It would be good to have `axios` configuration separated out into its own file, like a less complicated version of the following:

https://github.com/TTUSDC/cpceed-webapp/blob/master/src/server/core/connection.js